site/content/installation/ngf-images/pulling-ngf-image.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Pushing an NGINX Plus Gateway Fabric image to a private registry
+description: "This document describes how to pull a NGINX Plus Gateway Fabric image from the official F5 Docker registry and upload it to your private registry."
+weight: 100
```

Comment:
Can we change this weight to 300? Also, let's change the weight of `installing-ngf/_index.md` to 100, and `running-on-kind.md` to 400.

I'd like the order on the docs page to look like `Install`, `Expose`, `Images`, then `deploy on kind`.

Reply:
These aren't showing up in this order yet. Be sure to check out the netlify preview to ensure the order that the pages are displayed.

Reply:
This order is still wrong. Please check out the deploy preview. Should be `Install`, `Expose`, `Images`, then `deploy on kind`